README.md

```diff
@@ -2,6 +2,9 @@
 
 Backend application for [Artist Centered Image Hosting](https://github.com/fenya123/bingin).
 
+## System requirements
+
+1. Poetry package manager. Official installation guide: https://python-poetry.org/docs/#installation
```

Comment:
<details><summary>Opinionated suggestion</summary>
<p>


Let's leverage markdown links syntax for brevity's sake:
```md
1. [Poetry](https://python-poetry.org/docs/#installation) package manager.
```
To make it render to this:
> 1. [Poetry](https://python-poetry.org/docs/#installation) package manager.

***

Also I would suggest to omit "package manager" words, because I see in the future we will need to add docker here as well and it would be hard to keep consistency:
> 1. [Poetry](https://python-poetry.org/docs/#installation) package manager.
> 2. [Docker Desktop]() ???

***

And I would like to suggest to get rid of enumeration in this list, because there is no required order of the installation of those system requirements. We just expect them to be on developer's machine and it doesn't matter in which order they will be installed.

</p>
</details>